src/media/js/addon/constants.js

```diff
@@ -18,7 +21,30 @@ export const NOTE_TYPES = {
                                   color: 'gray'},
 };
 
+export const STATUS_INCOMPLETE = 'incomplete';
+export const STATUS_OBSOLETE = 'obsolete';
 export const STATUS_PENDING = 'pending';
 export const STATUS_PUBLIC = 'public';
-export const STATUS_OBSOLETE = 'obsolete';
 export const STATUS_REJECTED = 'rejected';
+
+
+export const ADDON_STATUS = {'disabled': gettext('Disabled')};
+ADDON_STATUS[STATUS_INCOMPLETE] = gettext('Incomplete');
+ADDON_STATUS[STATUS_OBSOLETE] = gettext('Obsolete');
+ADDON_STATUS[STATUS_PENDING] = gettext('Pending');
+ADDON_STATUS[STATUS_PUBLIC] = gettext('Public');
+ADDON_STATUS[STATUS_REJECTED] = gettext('Rejected');
+
+export const humanizeAddonStatus = (status, isDisabled) => {
```

Comment:
I was mirroring what you did [for categories](https://github.com/mozilla/marketplace-content-tools/blob/master/src/media/js/site/constants/categories.js#L28-L36).